staging/src/k8s.io/apimachinery/pkg/util/file/file.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package file
+
+import (
+	"os"
+	"testing"
+)
+
+// RemoveTestFile is a helper to close and remove test file.
+func RemoveTestFile(t *testing.T, f *os.File) {
```

Comment:
This is a really weird place to stick a test-only helper function.  Why is this right?

Reply:
Actually I' not sure where it should be placed, maybe we can find another choice here.

Reply:
Unfortunately, there are some cycle import dependency when putting this test-only func in pkg `k8s.io/kubernetes`, shall we put it back in pkg `k8s.io/apimachinery` and make a new testing dir?